### PR TITLE
chore: provide fmt.Stringer for EventType

### PR DESCRIPTION
**Pull request by smira**
Simple fix to allow easier printing of events.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

### CODE REVIEW

**Review by andrewrynhard — APPROVED** [review]

File: pkg/state/state.go [hunk 1]
```diff
@@ -24,6 +24,10 @@ const (
 	Destroyed
 )
 
+func (eventType EventType) String() string {
+	return [...]string{"Created", "Updated", "Destroyed"}[eventType]
```
Comment (AlekSi) [inline pkg/state/state.go:28]: Why not use `stringer` tool? https://pkg.go.dev/golang.org/x/tools/cmd/stringer

Comment (smira) [reply pkg/state/state.go:28]: never used, that, good point, if this requires changing build steps, might be too much work for this PR, but certainly something for the future

Comment (smira) [reply pkg/state/state.go:28]: https://github.com/talos-systems/kres/issues/53

**Pull request merged**